pkg/cli/plugins_test.go

```diff
@@ -27,88 +29,89 @@ var _ = Describe("resolvePluginsByKey", func() {
 
```

Comment:
the same scenario of https://github.com/kubernetes-sigs/kubebuilder/pull/1547/files#r435137335 (test descriptions are unreadable and do not allow we know easily what/when/why). See:  

Currenty, it is:
```go
Describe("resolvePluginsByKey", func() {
It("should resolve keys correctly", func() {
By("resolving foo.example.com/v1") // When? Why? 
By("resolving foo.example.com") //  When? Why?
```
 
IMO it should be such as:

```go
Describe("Checking plugin key resolution", func() {

It("should work successfully when is found a plugin that matches")
By(passing the name and version of the plugin)
By(passing just a valid plugin name)
By(passing just a valid plugin short name)

It("should fail when is not found a plugin that matches")
By(passing the name and version of the plugin)
By(passing just a valid plugin name)
By(passing just a valid plugin short name)

It("should fail when is found more than one plugin that matches")
By(passing the name and version of the plugin)
By(passing just a valid plugin name)
By(passing just a valid plugin short name)
```

Reply:
fair enough 👍